resources/dicts/patrols/new_cat_welcoming.json

```diff
@@ -2145,7 +2145,7 @@
         "max_cats": 6,
         "min_max_status": {},
         "weight": 20,
-        "intro_text": "The patrol comes across a o_c_n apprentice, waiting quietly at the border. When the apprentice notices the patrol, they press themselves to the ground, begging to join c_n.",
+        "intro_text": "The patrol comes across a o_c_n apprentice, waiting quietly at the border. When the apprentice notices the patrol, they press themself to the ground, begging to join c_n.",
```

Comment:
Shouldn't we use the Pronoun system?

Reply:
Hi - selkirks is right. We can use pronoun tagging for new cats, but only in patrol outcomes, rather than patrol intros (as the new cats are generated in the patrol outcome code blocks). I'd suggest a rewrite instead, such as "when the apprentice notices the patrol, the young cat presses close down to the ground, eyes round and beseeching, begging to join c_n." That avoids the use of pronouns for a cat who doesn't (code wise) exist yet